test_multiagent_fix.py

```
@@ -0,0 +1,92 @@
+    """Test multi-agent (should work after fix)"""
+    print("\n=== Testing Multi-Agent (should work after fix) ===")
+    try:
+        from praisonaiagents import Agent, Agents
+        
+        research_agent = Agent(instructions="Research about AI")
+        summarise_agent = Agent(instructions="Summarise research agent's findings")
+        agents = Agents(agents=[research_agent, summarise_agent])
+        result = agents.start("What is Python?")
+        print(f"✅ Multi-agent result: {result}")
+        return True
+    except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)